config/quality/pmd/pmd-ruleset.xml

```diff
@@ -20,8 +20,8 @@
     <rule ref="rulesets/java/naming.xml">
         <!--<exclude name="AbstractNaming" />-->
         <exclude name="LongVariable" />
-        <!--<exclude name="ShortMethodName" />-->
-        <!--<exclude name="ShortVariable" />-->
+        <exclude name="ShortMethodName" />
```

Comment:
I like the "redundant" version. It makes the method signature more explicit about what we're passing to it. A nice bonus, if you ask me.

As a last resort, we could simply [suppress the warning](http://pmd.sourceforge.net/snapshot/usage/suppressing.html).